### ISSUE TITLE
[Deque Analysis] File upload progress bar & success message lack color contrast

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1161]
## Deque Analysis Summary

1. While a file upload is in progress, the progress bar displayed has insufficient color contrast (e.g. "Queue length: 1 | sample.txt"):  Foreground: `#9C9C9C`, Background: `#FFFFFF`, The contrast ratio is: 2.7:1
2. After a file upload is successful (or really any successful task) a green success message appears.  This also has color contrast issues: Foreground: `#647D3E`, Background: `#E9F4D9` The contrast ratio is: 4.1:1

Deque ticket IDs: 469766, 471651

These are both most easily reproduced by drag & drop of file in Collection logo upload and/or submission.  Ideally, upload a large file to see the progress bar.

## Recommended Fix

For (1) likely darken the text to `#757575` (at a minimum)
For (2) likely make the green (foreground) text slightly darker, perhaps `#5A7336`

Both of these colors were found using https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by michdyk** [comment]
@tdonohue Could you assign me to this issue? My team and I have investigated the accessibility of the system recently, so I guess it will be great to have done this task until we are fresh after our initial analysis.

**Comment by michdyk** [comment]
@tdonohue please assign my colleague @Dawnkai to this task. He is going to provide the PR for this issue. Also when it will be accepted, I kindly ask to give credits for this bug fix contribution to PCG Academia. Thanks
